### PR TITLE
Docs: Work with metrics over time

### PR DESCRIPTION
**Pull request by validbeck**
## Internal Notes for Reviewers

> sc-6346

### Work with metrics over time

Net-new doc: [**LIVE PREVIEW**](https://docs-demo.vm.validmind.ai/pr_previews/beck/sc-6346/documentation-support-displaying-results/guide/monitoring/work-with-metrics-over-time.html)

### Other updates

Screenshot 2024-09-25 at 11 12 24 AM

- Added the "Work with metrics over time" page under `Ongoing monitoring`
- The default permissions for the "Ongoing monitoring" sections were incorrect. They are developer-locked (I tested this as I ran both the notebooks for unit metrics & ongoing monitoring as a developer-only role, then tried to follow the instructions as a validator-only role). 
- Made sure that the other content referencing content blocks for test-driven etc. had ongoing monitoring included where relevant.
- Cleaned up some of the awkward wording around instructions, especially in the "Enable monitoring" guide.

### CODE REVIEW

**Comment by validbeck** [comment]
@johnwalz97 Can you accuracy check this new doc for the explanation as to why metrics over time are relevant/used?

> https://docs-demo.vm.validmind.ai/pr_previews/beck/sc-6346/documentation-support-displaying-results/guide/monitoring/work-with-metrics-over-time.html

File: site/guide/monitoring/work-with-metrics-over-time.qmd [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: "Work with metrics over time"
+date: last-modified
+---
+
+Once generated via the {{< var validmind.developer >}}, view and add metrics over time to your ongoing monitoring plans in the {{< var validmind.platform >}}.
+
+Metrics over time refers to the continued monitoring of a model's performance once it is deployed. Tracking how a model performs as new data is introduced or conditions change ensures that it remains accurate and reliable in real-world environments where data distributions or market conditions shift. 
+
+- Model performance is determined by continously measuring metrics and comparing them over time to detect degration, bias, or shifts in the model's output.
```
Comment (johnwalz97) [inline site/guide/monitoring/work-with-metrics-over-time.qmd:10]:
```suggestion
- Model performance is determined by continuously measuring metrics and comparing them over time to detect degradation, bias, or shifts in the model's output. 
```

**Review by johnwalz97 — APPROVED** [review]
quick nitpick typo (sorry lol)... but other than that the content looks great to me 👍

File: site/guide/monitoring/work-with-metrics-over-time.qmd (hunk 1 above)
Comment (validbeck) [reply site/guide/monitoring/work-with-metrics-over-time.qmd]: NO need to apologise! Thank you so much for reviewing, and I should have caught that dur.

**Review by even-steven — APPROVED** [review]
Looks good 👍🏽

**Pull request merged**